blamo/lib/LogInfo/loginfo.dart

```diff
@@ -50,101 +71,206 @@ class _LogInfoPageState extends State<LogInfoPage> {
                 child: Column(
                 children: <Widget>[ 
                   FormBuilderTextField(
+                    textInputAction: TextInputAction.next,
                     attribute: 'title',
                     validators: [],
                     decoration: InputDecoration(labelText: "Project Name"),
-                  ),
-                  FormBuilderDateTimePicker(
-                    attribute: "start-date",
-                    inputType: InputType.date,
-                    validators: [],
-                    format: DateFormat("dd-MM-yyyy"),
-                    decoration: InputDecoration(labelText: "Start Date"),
-                  ),
-                  FormBuilderDateTimePicker(
-                    attribute: 'end-date',
-                    inputType: InputType.date,
-                    validators: [],
-                    format: DateFormat('dd-MM-yyyy'),
-                    decoration: InputDecoration(labelText: "End Date"),
+                    onFieldSubmitted: (v){
+                      FocusScope.of(context).requestFocus(formNodes[1]);
```

Comment:
Your Index is starting at 1. I printed out the Nodes and filled out the form. You are not saving the Project Name in the node.